docs/howtoguides/enable_realtime_kernel.rst

```diff
@@ -10,8 +10,8 @@ The `real-time kernel <realtime_>`_ is currently supported on Ubuntu
 22.04 LTS (Jammy) and 24.04 LTS (Noble). For more information, feel free to
 `contact our real-time team`_.
```

Comment:
This statement should be removed or changed so it doesn't refer to specific versions, as it makes it hard to maintain. See https://canonical-real-time-ubuntu-documentation.readthedocs-hosted.com/en/latest/how-to/enable-real-time-ubuntu/ which points to the releases reference document, which in future will also cover Ubuntu Core versions.